GUI/Utils/DarkModeForms/DarkModeCS.cs

```diff
@@ -0,0 +1,1037 @@
+                themeColors.WindowHighlight = Color.FromArgb(131, 236, 255);
+
+                themeColors.Container = Color.FromArgb(245, 169, 184);
+                themeColors.ContainerBorder = Color.FromArgb(255, 179, 194);
+                themeColors.ContainerHighlight = Color.FromArgb(185, 109, 124);
+
+                themeColors.Text = Color.White;
+                themeColors.TextInactive = Color.FromArgb(30, 30, 30);
+                themeColors.TextHighlight = Color.DodgerBlue;
+
+                themeColors.Control = Color.FromArgb(245, 169, 184);
+                themeColors.ControlBorder = Color.FromArgb(245, 169, 184);
+                themeColors.ControlHighlight = Color.FromArgb(255, 179, 194);
+
+                themeColors.Accent = Color.DodgerBlue;
+            }
+
+            if (Window != null)
+            {
+                ApplySystemTheme(Window);
+
+                Window.BackColor = themeColors.Window;
+                Window.ForeColor = themeColors.Text;
+            }
+
+            return themeColors;
+        }
+
+        /// <summary>Recolor image</summary>
+        /// <param name="bmp">Image to recolor</param>
+        /// <param name="c">Color</param>
+        public static Bitmap ChangeToColor(Bitmap bmp, Color c)
+        {
+            Bitmap bmp2 = new Bitmap(bmp.Width, bmp.Height);
+            using (Graphics g = Graphics.FromImage(bmp2))
+            {
+                g.InterpolationMode = InterpolationMode.HighQualityBilinear;
+                g.CompositingQuality = CompositingQuality.HighQuality;
+                g.SmoothingMode = SmoothingMode.HighQuality;
+
+                float tR = c.R / 255f;
+                float tG = c.G / 255f;
+                float tB = c.B / 255f;
+
+                System.Drawing.Imaging.ColorMatrix colorMatrix = new System.Drawing.Imaging.ColorMatrix(new float[][]
+                {
+                new float[] { 1,    0,  0,  0,  0 },
+                new float[] { 0,    1,  0,  0,  0 },
+                new float[] { 0,    0,  1,  0,  0 },
+                new float[] { 0,    0,  0,  1,  0 },  //<- not changing alpha
+				new float[] { tR,   tG, tB, 0,  1 }
+                });
+
+                System.Drawing.Imaging.ImageAttributes attributes = new System.Drawing.Imaging.ImageAttributes();
+                attributes.SetColorMatrix(colorMatrix);
+
+                g.DrawImage(bmp, new Rectangle(0, 0, bmp.Width, bmp.Height),
+                    0, 0, bmp.Width, bmp.Height, GraphicsUnit.Pixel, attributes);
+
+                attributes.Dispose();
+            }
+            return bmp2;
+        }
+        public static Image ChangeToColor(Image bmp, Color c) => (Image)ChangeToColor((Bitmap)bmp, c);
+
+        #endregion
+
+        #region Private Methods
+
+        private static bool DebugTheme;
+
+        private void ApplyTheme(Form _Form)
+        {
+            IsDarkMode = IsWindowsDarkThemed();
+            ThemeColors = GetAppTheme(_Form);
```

Comment:
Both of these should be called only once when the app starts instead of for every form.